src/Form/CocktailType.php

```diff
@@ -20,7 +21,9 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
     public function configureOptions(OptionsResolver $resolver): void
     {
         $resolver->setDefaults([
-            'data_class' => CocktailName::class,
+            'empty_data' => function (FormInterface $form) {
```

Comment:
What is this for? And why data_class is not needed anymore?

Reply:
fixed, forgot to change while testing